src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import roslib
+PACKAGE = "param_server"
```

Comment:
I *think* this should be `PACKAGE = "thunderbots"` (as that the project name specified in the `CMakeLists.txt` and the `package.xml` files), otherwise other nodes in this package will be looking under the wrong namespace (though I might be wrong about this).

Reply:
the package is the namespace the dynamic_reconfigure package puts the ParamsConfig stuff in. That can be anything, its only for internal use in the param_server node